### ISSUE TITLE
Header title path is truncated when short

### ISSUE THREAD

**Issue by hegelstad** [zeit/hyper#2641]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

**High Sierra 10.13.2** 
**Hyper 2.0.0-canary.11 (canary)** 
**.hyper.js Gist: [Gist](https://gist.github.com/hegelstad/f42b0c221ade2ae09bb88cc988d27c50/) 
**Relevant information from devtools**:
Title element in Developer Tools:
`<div class="tabs_title title_c9fzcr">..b/Marketplace</div>`
 

## Issue
Hi, I will let this picture speak for itself.

Title should not truncate "github" part of the path.

![image](https://user-images.githubusercontent.com/6997343/35473648-acb88cc0-0383-11e8-8fff-3e41b2be12e0.png)

**Comment by alejost848** [comment]
Same here, I'm on Windows and I'm facing the issue too

![hyper_2018-03-09_00-09-15](https://user-images.githubusercontent.com/8323249/37191515-27bf65d2-232e-11e8-8ffc-10b10f5eb533.png)

**Comment by setek** [comment]
Hi, any update on this? Or any quick clues where to look, and I'll see if I can put up a PR?

**Comment by LabhanshAgrawal** [comment]
@setek Is this issue still there? Don't see it on my end (macos 10.15, hyper 3.0.2).

**Comment by setek** [comment]
Hi @LabhanshAgrawal ugh my bad, sorry. Looks like it was an Oh My Zsh "feature" to pre-ellipsis it after a mere 13 chars! After setting my own window title, it's taking up available space. Thanks!

**Comment by LabhanshAgrawal** [comment]
@hegelstad 
from the last comment I guess this problem is not there anymore
can you confirm and close this issue

**Comment by hegelstad** [comment]
Works in single tab:

![image](https://user-images.githubusercontent.com/6997343/79561920-70979780-80aa-11ea-9ad4-0dabbc49107f.png)

Same problem in multiple tabs:

**Comment by LabhanshAgrawal** [comment]
> Hi @LabhanshAgrawal ugh my bad, sorry. Looks like it was an Oh My Zsh "feature" to pre-ellipsis it after a mere 13 chars! After setting my own window title, it's taking up available space. Thanks!

@hegelstad can you check if you're not facing the same pre-ellipsis thing. I checked the source code for the tabs and the ellipsis are done via css on text overflow. But there's space on your tab bar.
Also check if you're using any theme/plugins.
You can also try out the latest canary build from [ci](https://github.com/zeit/hyper/actions/runs/77488656) (there's been lot's of improvements since last release)

**Comment by LabhanshAgrawal** [comment]
now that I looked carefully you seem to be using some plugin to show the command+1 keyboard shortcut, try once after disabling it also.

**Comment by hegelstad** [comment]
Hey @LabhanshAgrawal I am really sorry :( I have shown you a picture of iTerm 2 with a theme that looks very similar to Hyper, so I got things mixed up.

I downloaded the canary version of hyper 3.1-4 and it checks out fine!

![image](https://user-images.githubusercontent.com/6997343/79570448-c162bc80-80b9-11ea-8402-d2534bd2af6c.png)

I remember why I gave up hyper and went back to iTerm 2:
https://www.loom.com/share/c0782ad83e9541359a6538ee27e7e028

These backticks and some other characters are appearing about 20pxs to the right when I trigger them, and then they move to the correct place when I write a new character or move the cursor with the keypad.
